vaccine/healthcheck_ussd.py

```diff
@@ -0,0 +1,796 @@
+    #   You get a `Event loop is closed` error
+    return aiohttp.ClientSession(
+        timeout=aiohttp.ClientTimeout(total=5),
+        headers={
+            "Authorization": f"Token {config.EVENTSTORE_API_TOKEN}",
+            "Content-Type": "application/json",
+            "User-Agent": "healthcheck-ussd",
+        },
+    )
+
+
+def get_google_api():
+    return aiohttp.ClientSession(
+        timeout=aiohttp.ClientTimeout(total=5),
+        headers={
+            "Content-Type": "application/json",
+            "User-Agent": "Jsbox/Covid19-Triage-USSD",
+        },
+    )
+
+
+def get_turn():
+    return aiohttp.ClientSession(
+        timeout=aiohttp.ClientTimeout(total=5),
+        headers={
+            "Authorization": f"Bearer {config.TURN_API_TOKEN}",
+            "Content-Type": "application/vnd.v1+json",
+            "User-Agent": "healthcheck-ussd",
+        },
+    )
+
+
+class Application(BaseApplication):
+    START_STATE = "state_start"
+
+    def calculate_risk(self):
+        answers = self.user.answers
+
+        symptom_count = 0
+        for answer in [
+            answers.get("state_fever"),
+            answers.get("state_cough"),
+            answers.get("state_sore_throat"),
+            answers.get("state_breathing"),
+            answers.get("state_taste_and_smell"),
+        ]:
+            if answer == "yes":
+                symptom_count += 1
+
+        if answers.get("confirmed_contact") == "yes":
+            if answers.get("state_province") == "ZA-WC":
+                if (
+                    int(answers.get("state_age_years")) > 55
+                    or answers.get("state_preexisting_conditions") == "yes"
+                ) and symptom_count >= 1:
+                    return "high"
+                return "moderate"
+            if symptom_count >= 1:
+                return "high"
+            return "moderate"
+
+        if symptom_count == 0:
+            if answers.get("state_exposure") == "yes":
+                return "moderate"
+            return "low"
+
+        if symptom_count == 1:
+            if answers.get("state_exposure") == "yes":
+                return "high"
+            return "moderate"
+
+        if symptom_count == 2:
+            if answers.get("state_exposure") == "yes":
+                return "high"
+            if answers.get("state_age") == ">65":
+                return "high"
+            return "moderate"
+
+        return "high"
+
+    def pad_location(self, location, places):
+        # Pads the integer part of the number to places
+        # Ensures that there's always a sign
+        # Ensures that there's always a decimal part
+        sign = "+"
+        if location < 0:
+            sign = "-"
+            location = location * -1
+
+        location = location.split(".")[2]
+        integer = str(location[0])
+        decimal = location[1] or 0
+
+        return f"{sign}{integer.zfill(places)}.{decimal}"
```

Comment:
This logic was from the javascript limitations, perhaps we can just reuse this logic instead? https://github.com/praekeltfoundation/healthcheckbot/blob/master/base/actions/actions.py#L78-L97